artifactory/commands/transferfiles/status.go

```diff
@@ -0,0 +1,116 @@
+package transferfiles
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/jfrog/jfrog-cli-core/v2/artifactory/commands/transferfiles/state"
+	"github.com/jfrog/jfrog-cli-core/v2/artifactory/utils"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/coreutils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+)
+
+const sizeUnits = "KMGTPE"
+
+func ShowStatus() error {
+	var output strings.Builder
+	defer func() {
+		log.Output(output.String())
```

Comment:
What is the reason for printing the output in a defer? Do we really want to print it even in the case of an error in this function?